### PR TITLE
Adding Fastlane files

### PR DESCRIPTION
**Pull request by gkassabli**
This PR adds fastlane files that will be utilised to create distributable iOS builds

linked to: https://github.com/spatialos/gdk-for-unity/pull/651

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
```
Comment (gkassabli) [inline fastlane/Fastfile:14]: All files are generated by fastlane. Fastfile is the only one that contains logic for now

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+default_platform(:ios)
+
+platform :ios do
+    desc "Description of what the lane does"
```
Comment (gkassabli) [inline fastlane/Fastfile]: oops, will fix this

File: fastlane/report.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
```
Comment (jessicafalk) [inline fastlane/report.xml]: probably shouldn't commit the report

File: fastlane/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+fastlane documentation
```
Comment (jessicafalk) [inline fastlane/README.md]: do we want this?
also, do we need to add some kind of license if we use fastlane?

**Comment by paulbalaji** [comment]
you'll need to change the branch name so that TeamCity picks it up

File: fastlane/Fastfile [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+platform :ios do
+    desc "Development build for iOS"
+    lane :dev do
+        cert(
+            development: true,
+            team_name: "Improbable Worlds Limited",
+        )
+        build_app(
+            scheme: "Unity-iPhone",
+            xcargs: "-allowProvisioningUpdates DEVELOPMENT_TEAM=\"27P8Y5FRY4\"",
```
Comment (jessicafalk) [inline fastlane/Fastfile:27]: should we have that key here?

File: fastlane/Appfile [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
+# apple_id("[[APPLE_ID]]") # Your Apple email address
+
+
+# For more information about the Appfile, see:
+#     https://docs.fastlane.tools/advanced/#appfile
```
Comment (paulbalaji) [inline fastlane/Appfile]: is this meant to be indented like that?

File: fastlane/Fastfile (hunk 5 above)
Comment (gkassabli) [reply fastlane/Fastfile:27]: This is apple team ID, which isn't secret and shouldn't change

File: fastlane/Appfile (hunk 6 above)
Comment (gkassabli) [reply fastlane/Appfile]: Generated code, let me update this

**Review by jessicafalk — APPROVED** [review]

**Pull request closed without merging**